dfx_generate.sh

```
@@ -0,0 +1,11 @@
+# this script is for quick re-generation of declarations. Replaces @dfinity imports with @icp-sdk
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)